docs/whatsnew/2.0.rst

```diff
@@ -10,7 +10,29 @@ Overview
 ========
 
 The SunPy project is pleased to announce the 2.0 release of the sunpy package.
-
+On this page, you can read about some of the big changes in this release:
+
+* :ref:`whatsnew-2.0-python`
+* :ref:`whatsnew-2.0-search`
+* :ref:`whatsnew-2.0-aiaprep`
+* :ref:`whatsnew-2.0-pixelindex`
+* :ref:`whatsnew-2.0-standards`
+* :ref:`whatsnew-2.0-overview`
+* :ref:`whatsnew-2.0-coordframe`
+* :ref:`whatsnew-2.0-carrington`
+* :ref:`whatsnew-2.0-proxy`
+* :ref:`whatsnew-2.0-citation`
```

Comment:
Why not just add a local table of contents?

Reply:
Good question.

Reply:
Is the plan to leave it like this, or add a contents?

Reply:
Yes.